dt-assets/functions/enqueue-scripts.php

```diff
@@ -337,7 +337,7 @@ function dt_site_scripts() {
         'save' => __( 'Save', 'disciple_tools' ),
         'edit' => __( 'Edit', 'disciple_tools' ),
         'delete' => __( 'Delete', 'disciple_tools' ),
-        'txt_info' => _x( 'Showing _START_ of _TOTAL_', 'just copy as they are: _START_ and _TOTAL_', 'disciple_tools' ),
+        'txt_info' => _x( 'Showing _START_ of _TOTAL_', 'Just copy as they are: _START_ and _TOTAL_', 'disciple_tools' ),
```

Comment:
This will trigger a re-translation. I don't think it is worth it.

Reply:
Ok - I changed it back.